deep_quoridor/src/quoridor.py

```diff
@@ -341,6 +341,9 @@ def __init__(
     def create_new(self):
         return Quoridor(self.board.create_new(), self.current_player, self.action_encoder, self._goal_rows)
 
+    def copy(self):
```

Comment:
```suggestion
    def clone(self):
```

Small nit: slightly better name like this?

Reply:
I chose copy because it's what the python standard library seems to use for similar cases. For example if you have a list `foo` with some some stuff in it, `foo.copy()` returns a new list with the same elements in the same order. I'm fine with clone too - how strongly do you feel about it?

Reply:
Very weakly. keep the copy.